addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -1,64 +1,87 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="web.PropertiesField" owl="1">
-        <div t-ref="properties" class="row align-items-start">
+        <div t-ref="properties" class="row align-items-start"
+            t-att-class="{'d-none': propertiesList.length === 0 &amp;&amp; !state.addButtonVisible}">
+            <t t-set="unfoldedSeparators" t-value="state.unfoldedSeparators"/>
             <t t-foreach="groupedPropertiesList" t-as="propertiesListGroup" t-key="propertiesListGroup_index">
+                <t t-set="foldable" t-value="!!propertiesListGroup.title"/>
+                <t t-set="isFolded" t-value="propertiesListGroup.name &amp;&amp; !unfoldedSeparators.includes(propertiesListGroup.name)"/>
```

Comment:
I was not a fan of this `and`, because it use pyjs which is terrible (you have a mix of python and js), but look like the JS team is using the `and` synthax so I'll change :)